.github/workflows/integration-tests-on-release.yaml

```diff
@@ -18,34 +18,43 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-versions : ['v1.15.12','v1.18.15','v1.21.0']
+        kubernetes-versions : ['v1.15.12','v1.16.15','v1.18.15','v1.19.7','v1.20.2']
```

Comment:
We are just testing 4 versions in master and release-1.7, right? That seems sufficient instead of 5 versions